bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -270,7 +271,7 @@ class ItcImagingTable(val parameters: ItcParametersProvider) extends ItcTable {
         case INSTRUMENT_GSAOI  =>
           new ItcGsaoiImagingTableModel(keys, uniqueConfigs, inputs, results)
 
-        case INSTRUMENT_GNIRS | INSTRUMENT_NIFS | INSTRUMENT_NIRI =>
+        case INSTRUMENT_NIRI =>
```

Comment:
NIFS doesn't support imaging (except for acquisition, which currently is not supported by ITC). This is why I removed it (having NIFS listed here was an error). However, I now think it would be better to explicitly list all instruments that support imaging and throw an error in the `case _` statement, or remove it, to make clear that the caller of this method should never call this code for unsupported instruments. This case is already handled further up in the call chain that's why unsupported instruments are not handled anymore down here, but I guess the assumption should be that somebody can still call this code for unsupported instruments.